exercises/common-lisp-1-a/.meta/example-1.lisp

```diff
@@ -0,0 +1,19 @@
+(defpackage pokemon-battle
```

Comment:
There are currently two different example implementations, that do look quite different, which is great! We strive to have three different example implementations, which can then be used as the starting points for the second part of the exercise. Is it feasible to have a third, different example?

Reply:
Certainly! I can work on one of those or try to source some ideas from others!

Reply:
Excellent!